File: 2022/01/02.ts
//const line = require('fs').readFileSync('example.txt').toString().split('\n');
const line = require('fs').readFileSync('input.txt').toString().split('\n');

function getCalories() {
    let foodCount = new Array()
    let elf = 0
    line.forEach((cal, idx, array) => {
        if (cal !== '') elf += parseInt(cal)

        if (cal === '' || idx === array.length - 1) {
            foodCount.push(elf)
            elf = 0
            return;
        }

    });
    foodCount.sort((a, b) => b - a)
    return foodCount[0] + foodCount[1] + foodCount[2]
}

console.log('Answer: ' + getCalories())

File: 2024/day02/02.py
levels = []
safe = 0

#with open('input.txt', 'r') as file:
with open('example.txt', 'r') as file:
    for report in file:
        levels = report.split()
        levels = list(map(int, levels))
        print((levels))


    
#print(safe)